bleak/backends/bluezdbus/scanner.py

```diff
@@ -231,13 +231,17 @@ def _handle_advertising_data(self, path: str, props: Device1) -> None:
         _service_data = {k: bytes(v) for k, v in props.get("ServiceData", {}).items()}
         _service_uuids = props.get("UUIDs", [])
 
+        # Get tx power data
+        tx_power = props.get("TxPower",None)
```

Comment:
The default of `get()` is already `None`.
```suggestion
        tx_power = props.get("TxPower")
```